index.bs

```diff
@@ -322,6 +318,19 @@ it should change the DNS-SD service name to a new value, indicating a new
 mechanism for metadata discovery.
 
 
+Computing the Agent Fingerprint {#computing-agent-fingerprint}
+-------------------------------
+
+The <dfn>agent fingerprint</dfn> of an agent is computed by following these
+steps:
+
+1. Compute the [[RFC7469#section-2.4|SKPI Fingerprint]] of the [=agent certificate=]
```

Comment:
Super nit: In some other areas of index.bs an HTML list is used for describing processes. Should we standardize on one or the other?

Reply:
Markdown will convert this into an HTML `<ol>` for us, unless it's nested inside a manually-added HTML element.